src/Forms/DateField.php

```diff
@@ -122,25 +115,27 @@ class DateField extends TextField
     protected $rawValue = null;
 
     /**
-     * Check if calendar should be shown on the frontend
+     * Use HTML5-based input fields (and force ISO 8601 date formats).
      *
+     * @var bool
+     */
+    protected $html5 = false;
```

Comment:
Hmmm yeah I guess for most frontend implementations, it'll be a better default (opt-out of HTML5 rather than opt-in). For the CMS, it'll definitely be a better default, since we have a polyfill there. And it would save devs remembering that when writing `getCMSFields()`. I'll change it, and update the changelog